v2/index.html

```diff
@@ -162,29 +163,31 @@ <h2>General Principles</h2>
     <h2>The Plaine & Easie Code</h2>
     <section>
         <h3>Clef</h3>
-        <div class="issue" data-number="32"></div>
         <aside class="note">
             UNIMARC field 036 $m; MARC21 field 031 $g
         </aside>
         <p>
-            The clef code is preceded by <code>%</code> and is three characters long.
+            The clef is a required field. Every encoding MUST include a clef.
         </p>
-        <div class="issue" data-number="33"></div>
-        <p>
-            The first character specifies the clef shape.
-        </p>
-
         <p>
-            The second character is <code>-</code> to indicate modern notation or <code>+</code> to indicate mensural
-            notation.
+            The clef code MUST be three characters long. The first character specifies the clef shape and MUST
+            be one of the values <code>G</code>, <code>g</code> (octave G), <code>C</code>, or <code>F</code>.
         </p>
         <p>
-            The third character (numerals 1-5) indicates the position of the clef on the staff, starting from the bottom
-            line.
+            The second character MUST be one of the characters <code>-</code> to indicate modern notation, or
+            <code>+</code> to indicate mensural notation. Certain features in the musical notation are dependent
+            on this value.
         </p>
         <p>
-            If the music is written for a transposing instrument, notate the incipit at sounding pitch.
```

Comment:
Yes, it was removed. 

It **may** still belong somewhere, but I felt this was more about the practice of encoding, rather than the specification of the code, and I'm trying to keep the two separate. 

I had envisioned something along the lines of #50, so if we don't want to lose this we should create an issue specifically mentioning transposing instrument guidelines for that document.